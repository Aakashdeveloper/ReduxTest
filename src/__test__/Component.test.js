import React from 'react';
import LatestNews from '../component/home/LatestNews';
import {create} from 'react-test-renderer';

describe('LatestNews Component Snapshot', () => {
    test("LatestNews",() => {
        let tree = create(<LatestNews/>);
        expect(tree.toJSON()).toMatchSnapshot();
    })
})
