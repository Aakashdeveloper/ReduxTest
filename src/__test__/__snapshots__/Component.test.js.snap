// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LatestNews Component Snapshot LatestNews 1`] = `
<div
  className="home-latest"
/>
`;
